core/src/OpenMPTarget/Kokkos_OpenMPTarget_ParallelFor_Team.hpp

```diff
@@ -22,6 +22,12 @@
 #include <Kokkos_Parallel.hpp>
 #include <OpenMPTarget/Kokkos_OpenMPTarget_Parallel.hpp>
 
+// Intel architectures prefer the classical hierarchical parallelism that relies
+// on OpenMP.
+#if defined(KOKKOS_ARCH_INTEL_GPU)
+#define KOKKOS_IMPL_HIERARCHICAL_INTEL_GPU
+#endif
```

Comment:
Is that something that it would make sense to define somewhere more "central" and include in print as part of the configuration?

Reply:
We can put that macro in a more central place. I will do that. It will be better since we need it in more than one file. 
But I don't think we need this in configuration printing since the user is not concerned (IMO) with how we implement hierarchical parallelism.